plugins/kotlin/src/main/java/org/quiltmc/quiltflower/kotlin/KotlinImportCollector.java

```diff
@@ -0,0 +1,62 @@
+package org.quiltmc.quiltflower.kotlin;
+
+import org.jetbrains.java.decompiler.main.collectors.ImportCollector;
+import org.jetbrains.java.decompiler.util.TextBuffer;
+
+import java.util.Arrays;
+
+public class KotlinImportCollector {
```

Comment:
I know that *I* was the one who wrote this class, but future work could be done on allowing subclasses of `ImportCollector` and making public more portions of its code to simplify and clean up this code